shared_model/cryptography/blob.hpp

```diff
@@ -55,6 +55,18 @@ namespace shared_model {
       bool operator==(const Blob &rhs) const override {
         return blob() == rhs.blob();
       }
+
+      /**
+       * Method perform transforming object to old-fashion blob_t format
+       * @tparam Blob - type of blob
+       * @return blob_t array with own data
+       * Design note: this method is deprecated and should be removed after
+       * migration to shared model in whole project
+       */
+      template <typename Blob>
```

Comment:
May be we could change typename name to make it different from the name of interface?